nautobot/wireless/tables.py

```diff
@@ -183,6 +183,13 @@ class Meta(BaseTable.Meta):
         )
 
 
+class DeviceGroupWirelessNetworkTable(ControllerManagedDeviceGroupWirelessNetworkAssignmentTable):
```

Comment:
Is there a better name for this class? It's not at all obvious to me the difference in intent between `ControllerManagedDeviceGroupWirelessNetworkAssignmentTable` and `DeviceGroupWirelessNetworkTable`